packages/web-components/fast-element/src/platform.ts

```diff
@@ -100,3 +100,31 @@ export function createTypeRegistry<TDefinition extends TypeDefinition>(): TypeRe
         },
     });
 }
+
+/**
+ * Creates a function capable of locating metadata associated with a type.
+ * @returns A metadata locator function.
+ * @internal
```

Comment:
Should this be marked @public?

Reply:
I wanted to keep this particular utility internal so we don't have to support it officially. It's really just here to reduce duplication needed by the attributes and the observables.